content/en/news/releases/1.6.x/announcing-1.6/index.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Change Notes	title: Change Notes
+description: Istio 1.6 release notes.
+weight: 10
+---
+
+## Traffic Management
+- ***Added*** experimental support for the Kubernetes Service APIs.
```

Comment:
in the google doc there is a link, can we preserve it? Probably the same applies many other places

Reply:
Will add all links from doc as a better starting place.